scripts/kata-install/Dockerfile

```diff
@@ -6,7 +6,7 @@ ADD 50-kata-remote configuration-remote.toml /files/
 
 RUN mkdir -p /scripts
 
-ADD osc-kata-install.sh osc-configs-script.sh osc-log-level.sh lib.sh /scripts/
+ADD osc-kata-install.sh osc-log-level.sh lib.sh /scripts/
```

Comment:
I don't really understand why merging two scripts into one makes things simpler. Isn't it clearer what each step does when they are separated? Aren't there cases where you would want to reconfigure without reinstalling?

Reply:
I removed most of the content from that script, so I thought moving it into the other one wouldn’t be a problem. I don’t think there are any cases that require reconfiguration. The two functions I moved to the other script will be removed anyway, since these config files will be part of the RPM package. Nevertheless, I think you’re right - it makes more sense to keep them in separate scripts.